src/deploy/scope_managementgroup.ts

```diff
@@ -34,11 +34,13 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
             stderr: (data: BufferSource) => {
                 core.error(data.toString());
             },
-            stdline: (data: string) => {
-                if (!data.startsWith("[command]"))
-                    commandOutput += data;
-                // console.log(data);
+            stdout: (data: BufferSource) => {
+                commandOutput += data.toString();
+                // console.log(data.toString());
             },
+            debug: (data: string) => {
```

Comment:
How do these debug stream help?

Reply:
these print the debug messages of the library if debug flag is set